src/lib/indexer/eventProcessor.ts

```diff
@@ -38,13 +38,13 @@ const parseIntegerOrThrow = (value: string): number => {
   return result
 }
 
-/* INFO uncomment if we decided to use attachments
 const attachmentPayload = (map: Map<string, string>, key: string): AttachmentRecord => ({
   type: 'insert',
   id: UUIDv4(),
   ipfs_hash: getOrError(map, key),
+  filename: 'reason',
+  size: 0,
```

Comment:
these should be `null` I think. We don't yet know the filename or size and that's ok. Note these come from the attachment upload on the other side and are stored in ipfs

Reply:
yes, this could be it actually!